src/test/compile-fail/impl-duplicate-methods.rs

```diff
@@ -11,7 +11,7 @@
 struct Foo;
 impl Foo {
     fn orange(&self){}
-    fn orange(&self){}   //~ ERROR error: duplicate method
+    fn orange(&self){}   //~ ERROR duplicate associated function
```

Comment:
Isn't this a method? (something that takes a `self` parameter)

I would expect this to say "duplicate method" here